setup/initialize_database.sh

```diff
@@ -1,9 +1,36 @@
 #!/bin/sh
-echo "Setting up mysql database."
+echo "Setting up mysql database..."
 sudo mysql -uroot < acm.sql
-echo "Setting up submissions and problems data."
-mkdir ../auacm/app/data
-cp data.zip ../auacm/app/data/data.zip
-cd ../auacm/app/data
-unzip data.zip
-rm data.zip
+echo "Setting up test database..."
+echo "DROP DATABASE IF EXISTS acm_test; CREATE DATABASE acm_test; USE acm_test;\n" | mysql -uroot < acm.sql
```

Comment:
This isn't working at all for me. The acm.sql file actually has the line `USE acm;`, so we should probably find a better way of setting it up.